### DISCUSSION TITLE
Trouble running npm run start

### DISCUSSION THREAD

**Discussion by krivoox** [juanm4/hexagonal-architecture-frontend#5] [Q&A]
Hello!
I was deploying the code from this repository and following the instructions didn't work for me.
I followed the instructions for react-ui, install my node_module but on doing npm run start I get this problem
![image](https://user-images.githubusercontent.com/52187511/173918452-4de5d88a-0ffb-494d-921b-e12c6dedd19b.png)

Install typescript globally and in the same project but it doesn't work for me.

Am I missing any other step to be able to execute it?
should I run npm install in the "hexagonal-architecture/" directory?

By the way, I liked your post!

**Comment by juanm4** [comment]
Hi, I was thinking of updating all the libraries of this project in order to make it works. So give me a few days to do that and I let you know when it is done.

**Reply by direstrepo24** [reply]
Hi, update and call in another new project, but it gives me the following error:, Failed to load resource: the server responded with a status of 404 (Not Found) http://localhost:5174/infrastructure/instances/productRepositoryFake

Thanks, your contribution is very important.

**Comment by rajasegar** [comment]
This works
https://github.com/juanm4/hexagonal-architecture-frontend/issues/4#issuecomment-1093013283
